Fix tctl doc (#527)
temporalio/documentation: docs/system-tools/tctl.md

Workflow, show Workflow history, and signal Workflow.

## Run the CLI

There are a few ways to run the CLI.

### Install locally with Brew

You can install `tctl` using [Homebrew](https://brew.sh/):

```bash
brew install tctl
```

### Run from docker-compose container

If you are running Temporal Server locally using [docker-compose](https://github.com/temporalio/docker-compose) you can run `tctl` from `temporal-admin-tools` container with:
```bash
docker exec temporal-admin-tools tctl
```
If you plan to run `tctl` command multiply times it might be a good idea to create an alias:
```bash
alias tctl="docker exec temporal-admin-tools tctl"
```
and then invoke `tctl` command as if it is installed locally. For example to describe default namespace:
```bash
tctl namespace describe
```

### Run from docker image

Also `tctl` can be used directly from the [admin-tools](https://hub.docker.com/r/temporalio/admin-tools) docker image:
- On Linux:
```bash
docker run --rm -it --entrypoint tctl --network host --env TEMPORAL_CLI_ADDRESS=localhost:7233 temporalio/admin-tools:1.11.1
```
- On macOS/Windows:
```bash

> docker run --rm -it --entrypoint tctl --env TEMPORAL_CLI_ADDRESS=host.docker.internal:7233 temporalio/admin-tools:1.11.1

Author: I tested Linux version but if someone on macOS can test it, it would be great!

```bash
# use AllowDuplicateFailedOnly option to start a Workflow
tctl workflow start --tq hello-world --wt Workflow --et 60 -i '"temporal"' --wid "<duplicated workflow id>" --wrp AllowDuplicateFailedOnly

# use AllowDuplicate option to run a workflow
tctl workflow run --tq hello-world --wt Workflow --et 60 -i '"temporal"' --wid "<duplicated workflow id>" --wrp AllowDuplicate
```

> You may also set this inside your workflow code with `WorkflowOptions.WorkflowIdReusePolicy`.

Reviewer: ```suggestion
You can also set this inside your Workflow code with `WorkflowOptions.WorkflowIdReusePolicy`.
```

#### Show Workflow history

```bash
tctl workflow show \
  -w 3ea6b242-b23c-4279-bb13-f215661b4717 \
  -r 866ae14c-88cf-4f1e-980f-571e031d71b0
```

Shortcut without `-w` and `-r` flag:

```bash
tctl workflow showid 3ea6b242-b23c-4279-bb13-f215661b4717 866ae14c-88cf-4f1e-980f-571e031d71b0
```

If the `run_id` is not provided, the latest run history for the workflow_id:

```bash
tctl workflow show \
  -w 3ea6b242-b23c-4279-bb13-f215661b4717
```

Shortcut:

```bash
tctl workflow showid 3ea6b242-b23c-4279-bb13-f215661b4717
```

> #### Show Workflow execution information

Reviewer: ```suggestion
#### Show Workflow Execution information
```

```bash
tctl workflow describe -w 3ea6b242-b23c-4279-bb13-f215661b4717 -r 866ae14c-88cf-4f1e-980f-571e031d71b0
# a shortcut of this is (without -w -r flag)
tctl workflow describeid 3ea6b242-b23c-4279-bb13-f215661b4717 866ae14c-88cf-4f1e-980f-571e031d71b0

# if run_id is not provided, it will show the latest workflow execution of that workflow_id
tctl workflow describe -w 3ea6b242-b23c-4279-bb13-f215661b4717
# a shortcut of this is
tctl workflow describeid 3ea6b242-b23c-4279-bb13-f215661b4717
```

> #### Query Workflow execution

Reviewer: ```suggestion
#### Query Workflow Execution
```

```bash
# use custom query type
tctl workflow query -w <wid> -r <rid> --qt <query-type>

# use build-in query type "__stack_trace" which is supported by Temporal SDK
tctl workflow query -w <wid> -r <rid> --qt __stack_trace
# a shortcut to query using __stack_trace is (without --qt flag)
tctl workflow stack -w <wid> -r <rid>
```

#### Signal, cancel, terminate Workflow

```bash
# signal
tctl workflow signal -w <wid> -r <rid> -n <signal-name> -i '"signal-value"'

# cancel
tctl workflow cancel -w <wid> -r <rid>

# terminate
tctl workflow terminate -w <wid> -r <rid> --reason
```

> Terminating a running Workflow execution will record a WorkflowExecutionTerminated event as the closing event in the history.

Reviewer: ```suggestion
Terminating a running Workflow Execution records a WorkflowExecutionTerminated event as the closing event in the history.
```

> If you are familiar with the Temporal history event, You can also reset to any command finish event by using:

Reviewer: ```suggestion
If you are familiar with the Temporal history event, you can also reset to any command finish event by using:
```

> The command will store the bad binary checksum into namespace info and trigger a process to reset all your workflows.

Reviewer: ```suggestion
The command stores the bad binary checksum into namespace info and triggers a process to reset all your Workflows.
```

```bash
tctl --ns <YourNamespace> namespace update --add_bad_binary aae748fdc557a3f873adbe1dd066713f  --reason "rollback bad deployment"
```

> As you add the bad binary checksum to your namespace, Temporal will not dispatch any command tasks to the bad binary.

Reviewer: ```suggestion
After you add the checksum of the bad binary to your namespace, Temporal will not dispatch any command tasks to the bad binary.
```

  --query "WorkflowType='main.SampleParentWorkflow' AND ExecutionStatus='Running'"
```

> This will return all open Workflows with `workflowType` as `main.SampleParentWorkflow`:

Reviewer: ```suggestion
This returns all open Workflows with `workflowType` as `main.SampleParentWorkflow`.
```

> After the second start list Workflows with:

Reviewer: ```suggestion
After the second start, list Workflows with:
```
